### PR TITLE
mlflow integration command

### PR DESCRIPTION
**Pull request by ExtinctWolf83**
Added function to provide code for mlflow integration

### CODE REVIEW

File: deploifai/mlflow/get_setup.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+from deploifai.context import pass_deploifai_context_obj, DeploifaiContextObj
+from PyInquirer import prompt
+
+
+@click.command('get-setup')
+@click.option("--external", is_flag=True, help="Use code externally")
+@pass_deploifai_context_obj
+def get_setup(context: DeploifaiContextObj, external):
+    """
+    Command to obtain code for mlflow integration
```
Comment (98sean98) [inline deploifai/mlflow/get_setup.py:14]: Simplify the doc string a little. Try `Get setup to integrate mlflow in training scripts`.

File: deploifai/mlflow/get_setup.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                        }
+                        
+                    }
+                    """
+        project_data = context.api.get_project(
+            project_id=project_id, fragment=fragment
+        )
+        project_name = project_data["name"]
+        workspace_name = project_data["account"]["username"]
+        experiment_data = project_data["experiment"]
```
Comment (98sean98) [inline deploifai/mlflow/get_setup.py:40]: should be `project_data['experiments']`

File: deploifai/mlflow/get_setup.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        # elif len(experiment_names) == 1:
+        # experiment_name = experiment_names[0]
+        else:
+            choose_experiment = prompt(
+                {
+                    "type": "list",
+                    "name": "experiment",
+                    "message": "Choose a experiment",
+                    "choices": [
+                        exp for exp in experiment_names
```
Comment (98sean98) [inline deploifai/mlflow/get_setup.py:57]: There should be a better way to write this. Look at [cloud profile prompt](https://github.com/deploifai/cli/blob/077a9c612c33f15b6087e9ffc784234c674ad95d/deploifai/project/create.py#L113) from the project create command.

File: deploifai/mlflow/get_setup.py [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+        else:
+            required_code: str = line4+line5+line6
+
+        click.secho("The following code needs to be added for mlflow integration", fg='green')
+        click.secho(required_code)
+        pyperclip.copy(required_code)
+        click.secho("The code has been copied to your clipboard", fg='green')
+    else:
+        click.secho("Please run code in project directory")
```
Comment (98sean98) [inline deploifai/mlflow/get_setup.py:93]: Try a `fg='red'` here with `Unable to get project information (could not find .deploifai directory)`.

Later we're gonna unify this project information check with a click command decorator that we can write ourselves.

File: deploifai/mlflow/get_setup.py [hunk 5]
```diff
@@ -0,0 +1,93 @@
+import click
+import pyperclip
+
+from deploifai.api import DeploifaiAPIError
+from deploifai.context import pass_deploifai_context_obj, DeploifaiContextObj
+from PyInquirer import prompt
+
+
+@click.command('get-setup')
+@click.option("--external", is_flag=True, help="Use code externally")
```
[the remark this answers is not in the document]
Comment (ExtinctWolf83) [reply deploifai/mlflow/get_setup.py:10]: yeah
i think that makes more sense
will change

File: deploifai/mlflow/get_setup.py (hunk 3 above)
Comment (ExtinctWolf83) [reply deploifai/mlflow/get_setup.py:57]: That is where I got the idea for the prompt
What do you think should be changed to make it better?

File: deploifai/mlflow/get_setup.py (hunk 1 above)
Comment (ExtinctWolf83) [reply deploifai/mlflow/get_setup.py:14]: Changed

**Pull request merged**